inlanecruising_plugin/package.xml

```diff
@@ -36,10 +36,9 @@
   <depend>carma_wm</depend>
   <depend>basic_autonomy</depend>
   <depend>tf2</depend>
-  <depend>tf2_geometry_msgs</depend> 
-  <depend>trajectory_utils</depend>   
```

Comment:
References should also be removed from headers

Reply:
removed the references